toke.c

```diff
@@ -2054,7 +2055,6 @@ static int
 S_postderef(pTHX_ int const funny, char const next)
 {
     assert(funny == DOLSHARP
-        || memCHRs("$@%&*", funny)
```

Comment:
This removal seems interesting - Do we have any historical insight into why the test is done both with `memCHRs` and with the `==` tests below it? They seem to be duplicates

Reply:
Historically perly tokens where characters. Recently I replaced those used in perly with `PERLY_*` (to be able to distinguish between actual character literal and return value in toke). I did it one by one and after last one I just forgot to remove now obsoleted test.